src/translations.h

```diff
@@ -19,6 +19,10 @@
 #define translate_marker_context(c, x) x
 #endif
 
+#include <string>
+#include "json.h"
```

Comment:
Don't include "json.h" here, it's quite large and has more dependencies. You don't need to derive from `JsonDeserializer`, having a `deserialze` function with proper signature is enough.